src/applications/gqrx/mainwindow.cpp

```diff
@@ -668,8 +670,8 @@ void MainWindow::updateFrequencyRange(bool ignore_limits)
         qDebug() << QString("New frequnecy range: %1 - %2 MHz (step is %3 Hz but we use 1 Hz).").
                     arg(startd*1.0e-6).arg(stopd*1.0e-6).arg(stepd);
 
-        qint64 start = (qint64)startd + d_lnb_lo;
-        qint64 stop  = (qint64)stopd  + d_lnb_lo;
+        qint64 start = (qint64)(startd + rx->get_filter_offset()) + d_lnb_lo;
+        qint64 stop  = (qint64)(stopd + rx->get_filter_offset())  + d_lnb_lo;
```

Comment:
Ok, I think this is where the bug in question is but using the filter offset here is not correct either. I think we want:

qint64 start = (qint64)(startd - 0.5 * rx->get_input_rate()) + d_lnb_lo;
qint64 stop  = (qint64)(stopd + 0.5 * rx->get_input_rate())  + d_lnb_lo;

No?